src/pages/Dashboard/Dashboard.tsx

```diff
@@ -45,10 +45,10 @@ const DashboardPage: React.FC<AllProps> = () => {
         <h1 className='board-title'>Dashboard</h1>
       </PageSection>
       <PageSection>
-        <PageSection className='sectionArea' variant={PageSectionVariants.light}>
+        <PageSection className='section-area' variant={PageSectionVariants.light}>
```

Comment:
```suggestion
        <PageSection className="section-area" variant={PageSectionVariants.light}>
```

Reply:
Great suggestion! The practice for attribute values is indeed to use double quote marks.

Let's try to stick to double quote marks for new code and we can refactor the old code to have double quote marks later.